### DISCUSSION TITLE
Using basic logging instead of json format logging for unix file redirection

### DISCUSSION THREAD

**Discussion by dhruvmehtad11** [apollographql/router#2875] [Q&A]
I am redirecting the logs of apollo router to log file using rsyslog and separating the log files into out.log and err.log based on syslogseverity. But rsyslog is unable to identify difference between info and error log because of json formatted logs. Is there anyway to use basic logging instead of json format logging

**Comment by bnjjj** [comment]
I don't know if it's what you're asking for but [here in our documentation](https://www.apollographql.com/docs/router/configuration/logging#basic-configuration)  you can configure to format to `pretty` and it will be the text format. Let me know if it helps
